CONTRIBUTION.md

```diff
@@ -0,0 +1,47 @@
+# Contribution guidlines
+## Preamble
+These guidelines are intended as "guidelines" to how an PR To Dire's Buildinggadgets should be structures. 
+Bear in mind that these are only guidelines and don't necessarily apply in each and every case, but if you choose not to follow a specific Part of this specification, you must be able to reason out why, if asked to do so.
+## PR structure
+### General Notes
+In General a PR should explain what it does, how it does this and how it was tested. It should also follow the [Code Guidelines](#code-guidelines).
+* If it is a Bug fix, prepend it with \[Bug]/\[Fix#xyz].
```

Comment:
```suggestion

```
I'm ok with [WIP], but it's clear enough when something is a bug (not to mention the fact that people often think something is a bug, when it isn't -- or it's more of an oversight than a bug). I previously removed all the instances of these and replaced them with labels.

Ideally, you could set certain labels to be applicable by the person that opened a given issue/PR. Then I would make WIP a label; but sense that's not possible, I'm fine with allowing it as a [x] pseudo-label. Anything that can be an actual label should be.